javagen/src/main/java/com/azure/autorest/mapper/ClientMapper.java

```diff
@@ -292,6 +295,21 @@ private ClientResponse parseResponse(Operation method, JavaSettings settings) {
         return builder.build();
     }
 
+    private static ModuleInfo moduleInfo() {
+        // WARNING: Only tested for low level clients
+        JavaSettings settings = JavaSettings.getInstance();
+        ModuleInfo moduleInfo = new ModuleInfo(settings.getPackage());
+
+        List<ModuleInfo.RequireModule> requireModules = moduleInfo.getRequireModules();
+        requireModules.add(new ModuleInfo.RequireModule("com.azure.core", true));
+        requireModules.add(new ModuleInfo.RequireModule("com.azure.core.experimental", true));
+
+        List<ModuleInfo.ExportModule> exportModules = moduleInfo.getExportModules();
+        exportModules.add(new ModuleInfo.ExportModule(settings.getPackage()));
```

Comment:
If the LLC client is provided with a package name that has `implementation` in it, we should not export it. Should we even support generating LLC if the package name has `implementation`?

Reply:
If --low-level-client is turned on, the only 3 files we generate are the client, the builder, and the module-info. I don't quite understand why someone would put the client and builder in `implementation`?